services/discovery/Makefile

```diff
@@ -19,6 +19,14 @@ BUILD_DIR = .aws-sam
 DEPLOYMENT_FILE = .deploy
 
 
+####################
```

Comment:
Tested this Guard Default pattern with @mattyellen. It works pretty nicely. Go go `Make`. 😉

Reply:
looks good to me!